reactnative/ReactNativeTemplateApp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'android'
 
 dependencies {
-    compile project(':libs:SmartSync')
+    compile project(':libs:SalesforceSDK')
```

Comment:
I think we should include it by default. Otherwise we might need to support/document adding it after the fact.